packages/renderer/src/modules/store/snackbar/types.ts

```diff
@@ -1,8 +1,10 @@
 import type { AlertColor } from 'decentraland-ui2';
+import type { Project } from '/shared/types/projects';
 
 export type Severity = AlertColor | 'loading';
 export type NotificationId = string;
 type CustomNotificationType = 'missing-scenes';
+type DependencyNotificationType = 'new-dependency-version' | 'dependency-updated-automatically';
```

Comment:
I think this is not following the pattern already implemented for the snackbar. It has generic notifications (severity + message) and custom notification (everything else).

The `'new-dependency-version'` and `'dependency-updated-automatically'` should be added as custom notifications instead of creating a third type for them, ie:

```ts
type CustomNotificationType = 'missing-scenes' |  'new-dependency-version' | 'dependency-updated-automatically'
```

This also applies to the `createDependencyNotification` action/reducer (should use `createCustomNotification`)

Or is there a reason they have been separated that I'm missing?

If it is because of the filter to avoid showing duplicate notifications for the same project, that could be added in the `createCustomNotification` reducer.